### ISSUE TITLE
build/release github workflow

### ISSUE THREAD

**Issue by zeutyx** [sqids/sqids-postgresql#3]
Could you please implement a github workflow to build and release binaries for all 3 platforms. I'm a windows user, and it's really difficult to build it on my machine as pgrx isn't supported on windows, so I'd really appreciate a prebuilt binary. For now, I'll be using sqids-plpgsql, although I'm guessing it's slower due to the complied nature of this one.

**Comment by 4kimov** [comment]
Good idea, but looks like a few blockers on this one (https://github.com/pgcentralfoundation/pgrx/issues/1927). On pause for now.

**Comment by ayZagen** [comment]
@4kimov it seems the blocking issue is closed. Are there any plans to have prebuilt binaries?
